readthedocs/gold/forms.py

```diff
@@ -88,6 +89,18 @@ def __init__(self, *args, **kwargs):
         self.projects = kwargs.pop('projects', None)
         super(GoldProjectForm, self).__init__(*args, **kwargs)
 
+    def clean_project(self):
+        cleaned_data = super(GoldProjectForm, self).clean()
```

Comment:
You actually don't need to call to this method here, you can just use cleaned_data like this

https://github.com/rtfd/readthedocs.org/blob/90b1e52cfd6a124bf7298a1b3cb1711fddd67ab8/readthedocs/core/forms.py#L59-L65

Reply:
Wokring on it.